common/src/main/kotlin/entity/DiscordMessage.kt

```diff
@@ -114,6 +121,26 @@ public data class DiscordMessage(
     val thread: Optional<DiscordChannel> = Optional.Missing()
 )
 
+internal object NonceAsStringSerializer : KSerializer<String> {
```

Comment:
I've implemented something similar [here](https://github.com/kordlib/kord/blob/0.8.x/common/src/main/kotlin/entity/AuditLog.kt#L319). I suggest a refactor to use the same serializer in both places.
A good place to put this serializer would be [here](https://github.com/kordlib/kord/tree/0.8.x/common/src/main/kotlin/serialization).